ddtrace/bootstrap/sitecustomize.py

```diff
@@ -243,15 +243,12 @@ def _(threading):
         else:
             log.debug("additional sitecustomize found in: %s", sys.path)
 
-    if asbool(os.environ.get("DD_REMOTE_CONFIGURATION_ENABLED", "true")):
+    if config._remote_config_enabled:
```

Comment:
Just a memento, unrelated to this PR: it would be good to assess whether we can remove this variable entirely in 2.x

Reply:
@Yun-Kim ^